### PR TITLE
[FLINK-23222]Translate page 'Application Profiling & Debugging' into Chinese

### PR DESCRIPTION
**Pull request by pierre94**
## What is the purpose of the change

Translate "Application Profiling & Debugging" pages into Chinese.
The page url is `https://ci.apache.org/projects/flink/flink-docs-master/zh/docs/ops/debugging/application_profiling`.

## Brief change log

translate ` flink/docs/content.zh/docs/ops/debugging/application_profiling.md`

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (no)
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: (no)
  - The serializers: (no)
  - The runtime per-record code paths (performance sensitive): (no)
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn, ZooKeeper: (no)
  - The S3 file system connector: (no)

## Documentation

  - Does this pull request introduce a new feature? (no)

### CODE REVIEW

File: docs/content.zh/docs/ops/debugging/application_profiling.md [hunk 1]
```diff
@@ -25,60 +25,48 @@ specific language governing permissions and limitations
...
-If you encounter `OutOfMemoryExceptions` with your Flink application, then it is a good idea to enable heap dumps on out of memory errors.
+如果你的 Flink 应用程序遇到 `OutOfMemoryExceptions` ，那么启用在内存溢出错误时堆转储是一个好主意。
 
 ```yaml
 env.java.opts: "-XX:+HeapDumpOnOutOfMemoryError -XX:HeapDumpPath=${FLINK_LOG_PREFIX}.hprof"
 ```
 
-The heap dump will allow you to analyze potential memory leaks in your user code.
-If the memory leak should be caused by Flink, then please reach out to the [dev mailing list](mailto:dev@flink.apache.org).
+堆转储(Heap Dump)将使你能够方便地分析用户代码中潜在的内存泄漏问题。如果内存泄漏是由 Flink 引起的，那么请联系[开发人员邮件列表](mailto:dev@flink.apache.org)。
```
Comment (RocMarshal) [inline docs/content.zh/docs/ops/debugging/application_profiling.md:58]:
```suggestion
堆转储（Heap Dump）将使你能够方便地分析用户代码中潜在的内存泄漏问题。如果内存泄漏是由 Flink 引起的，那么请联系[开发人员邮件列表](mailto:dev@flink.apache.org)。
```

File: docs/content.zh/docs/ops/debugging/application_profiling.md [hunk 2]
```diff
@@ -25,60 +25,48 @@ specific language governing permissions and limitations
...
-[JITWatch](https://github.com/AdoptOpenJDK/jitwatch/wiki) is a log analyser and visualizer for the Java HotSpot JIT
-compiler used to inspect inlining decisions, hot methods, bytecode, and assembly. Example configuration:
+[JITWatch](https://github.com/AdoptOpenJDK/jitwatch/wiki) Java HotSpot JIT 编译器的日志分析器和可视化工具，用于检查内联决策，热方法，字节码和汇编。配置示例：
 
 ```yaml
 env.java.opts: "-XX:+UnlockDiagnosticVMOptions -XX:+TraceClassLoading -XX:+LogCompilation -XX:LogFile=${FLINK_LOG_PREFIX}.jit -XX:+PrintAssembly"
 ```
 
-## Analyzing Out of Memory Problems
+## 分析内存溢出(OOM)问题
```
Comment (RocMarshal) [inline docs/content.zh/docs/ops/debugging/application_profiling.md:50]:
```suggestion
## 分析内存溢出（OOM）问题
```

File: docs/content.zh/docs/ops/debugging/application_profiling.md [hunk 3]
```diff
@@ -25,60 +25,48 @@ specific language governing permissions and limitations
...
-If the memory leak should be caused by Flink, then please reach out to the [dev mailing list](mailto:dev@flink.apache.org).
+堆转储(Heap Dump)将使你能够方便地分析用户代码中潜在的内存泄漏问题。如果内存泄漏是由 Flink 引起的，那么请联系[开发人员邮件列表](mailto:dev@flink.apache.org)。
 
-## Analyzing Memory & Garbage Collection Behaviour
+## 分析内存和垃圾回收行为
 
-Memory usage and garbage collection can have a profound impact on your application.
-The effects can range from slight performance degradation to a complete cluster failure if the GC pauses are too long.
-If you want to better understand the memory and GC behaviour of your application, then you can enable memory logging on the `TaskManagers`.
+内存使用和垃圾回收会对你的应用程序产生巨大的影响。如果 GC 停顿时间过长，其影响力小到性能下降，大到集群全面瘫痪。如果你想更好地理解应用程序的内存和 GC 行为，可以在 `TaskManagers` 上启用内存日志记录
```
Comment (RocMarshal) [inline docs/content.zh/docs/ops/debugging/application_profiling.md:62]:
```suggestion
内存使用和垃圾回收会对你的应用程序产生巨大的影响。如果 GC 停顿时间过长，其影响小到性能下降，大到集群完全瘫痪。如果你想更好地理解应用程序的内存和 GC 行为，可以在 `TaskManagers` 上启用内存日志记录。
```

**Review by RocMarshal — COMMENTED** [review]
@pierre94 Thank you for your contribution. I left a few comments. Please let me know what do you think of it.

**Comment by pierre94** [comment]
@RocMarshal tks, i think your suggestions are better. I have updated the PR.

File: docs/content.zh/docs/ops/debugging/application_profiling.md [hunk 4]
```diff
@@ -25,60 +25,48 @@ specific language governing permissions and limitations
...
 ```yaml
 env.java.opts: "-XX:+UnlockCommercialFeatures -XX:+UnlockDiagnosticVMOptions -XX:+FlightRecorder -XX:+DebugNonSafepoints -XX:FlightRecorderOptions=defaultrecording=true,dumponexit=true,dumponexitpath=${FLINK_LOG_PREFIX}.jfr"
 ```
 
-## Profiling with JITWatch
+## 使用 JITWatch 分析
 
-[JITWatch](https://github.com/AdoptOpenJDK/jitwatch/wiki) is a log analyser and visualizer for the Java HotSpot JIT
-compiler used to inspect inlining decisions, hot methods, bytecode, and assembly. Example configuration:
+[JITWatch](https://github.com/AdoptOpenJDK/jitwatch/wiki) Java HotSpot JIT 编译器的日志分析器和可视化工具，用于检查内联决策，热方法，字节码和汇编。配置示例：
```
Comment (RocMarshal) [inline docs/content.zh/docs/ops/debugging/application_profiling.md:44]: What about `热方法` -> `热点方法`？

Comment (pierre94) [reply docs/content.zh/docs/ops/debugging/application_profiling.md:44]: `热点方法` is more professional

Comment (pierre94) [reply docs/content.zh/docs/ops/debugging/application_profiling.md]: I have updated again.

File: docs/content.zh/docs/ops/debugging/application_profiling.md [hunk 5]
```diff
@@ -25,60 +25,48 @@ specific language governing permissions and limitations
...
 
 ```yaml
 env.java.opts: "-XX:+UnlockDiagnosticVMOptions -XX:+TraceClassLoading -XX:+LogCompilation -XX:LogFile=${FLINK_LOG_PREFIX}.jit -XX:+PrintAssembly"
 ```
 
-## Analyzing Out of Memory Problems
+## 分析内存溢出（OOM）问题
 
-If you encounter `OutOfMemoryExceptions` with your Flink application, then it is a good idea to enable heap dumps on out of memory errors.
+如果你的 Flink 应用程序遇到 `OutOfMemoryExceptions` ，那么启用在内存溢出错误时堆转储（Heap Dump）是一个好主意。
```
Comment (RocMarshal) [inline docs/content.zh/docs/ops/debugging/application_profiling.md:52]:
```suggestion
如果你的 Flink 应用程序遇到 `OutOfMemoryExceptions`，那么启用在内存溢出错误时堆转储（Heap Dump）是一个好主意。
```

Comment (pierre94) [reply docs/content.zh/docs/ops/debugging/application_profiling.md]: Updated again.

**Review by RocMarshal — APPROVED** [review]
Thank you @pierre94 for your effort.  LGTM.  Now ping @wuchong .

**Comment by wuchong** [comment]
Thanks @pierre94 for the great work and @RocMarshal  for the reviewing. 
Merging...

**Pull request merged**